tests/nnstreamer_converter/unittest_converter.cc

```diff
@@ -395,11 +395,11 @@ TEST (tensorConverterPython, dynamicDimension)
   data_received = (guint *) g_malloc0 (sizeof (guint));
   g_signal_connect (sink_handle, "new-data", (GCallback)new_data_cb, data_received);
 
-  buf_0 = gst_buffer_new_wrapped (g_memdup (_test_frames1, 24), 24);
-  buf_1 = gst_buffer_new_wrapped (g_memdup (_test_frames2, 48), 48);
+  buf_0 = gst_buffer_new_wrapped (g_memdup (_test_frames1, 96), 96);
+  buf_1 = gst_buffer_new_wrapped (g_memdup (_test_frames2, 192), 192);
   buf_2 = gst_buffer_copy (buf_0);
 
-  EXPECT_EQ (setPipelineStateSync (pipeline, GST_STATE_PLAYING, UNITTEST_STATECHANGE_TIMEOUT), 0);
+  EXPECT_EQ (setPipelineStateSync (pipeline, GST_STATE_PLAYING, TEST_TIMEOUT_MS), 0);
```

Comment:
no need to change for custom converter? (#L116)

Reply:
The Python custom converter took a long time, so I changed it.
I also changed the custom code type converter test to prevent unnecessary test failure.
Thanks.